applications/mne_rt_server/connectors/Neuromag/dacqserver.h

```diff
@@ -42,49 +41,44 @@
 // INCLUDES
 //=============================================================================================================
 
-#include "types_definitions.h"
-#include <fiff/fiff_info.h>
-#include <fiff/fiff_tag.h>
-
 
 //*************************************************************************************************************
 //=============================================================================================================
 // QT INCLUDES
 //=============================================================================================================
 
 #include <QThread>
-
-#include <QTcpSocket>
-
-#include <QByteArray>
+#include <QSharedPointer>
+#include <QPointer>
 
 
 //*************************************************************************************************************
 //=============================================================================================================
-// DEFINE NAMESPACE NeuromagPlugin
+// FORWARD DECLARATIONS
```

Comment:
Isn't it the namespace definition?

Reply:
No these are the forward declarations. The namespace declarations follows below.